usr.bin/xinstall/tests/install_test.sh

```diff
@@ -497,6 +497,30 @@ set_owner_group_mode_unpriv_body() {
 	atf_check_equal "$u:$g:10$cM" "$(stat -f"%u:%g:%p" testc)"
 }
 
+atf_test_case metalog_unpriv
+metalog_unpriv_head() {
+	atf_set "require.user" "unprivileged"
+}
+metalog_unpriv_body() {
+	printf "test" >testf
+
+	atf_check install -U -M METALOG -o root -g wheel -f arch testf testc
+	atf_check grep -q "testc .*uname=root gname=wheel .*flags=arch" METALOG
+
+	rm -f METALOG testc
+	atf_check -s exit:1 -e match:"unknown user no-such-user" install -U -o no-such-user testf testc
```

Comment:
I thought this was a feature, especially if used together with `-M`. That is, `-U` and `-o` without `-M` should be a NOP?